### DISCUSSION TITLE
Duration Limit

### DISCUSSION THREAD

**Discussion by yoshi-1012** [drand/tlock#81] [Q&A]
Is there a limit on the duration for the tool?

Can you do a timelock for over 1 year?

**Comment by AnomalRoil** [comment]
No specific limits, but it requires that the decryption network (the League
of Entropy, e.g.) is still up and running by then.
1y or 2y should be fine with the current network, but because the
cryptography isn't quantum resistant, 10y or more is probably too far away,
since we might have migrated to post quantum options by then.
